### PR TITLE
Adds disclaimer for contributions 

### PR DESCRIPTION
**Pull request by iteles**
As discussed and agreed in #36, adding a temporary _strongly worded_ disclaimer to ensure we maintain the integrity of this repo. :+1:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,27 @@
 # 💬 `quotes`
 
+### Disclaimer for contributions
+
+If you are thinking of _contributing_ to this repository (hooray! :tada:),
```
Comment (nelsonic) [inline README.md:5]:
```suggestion
If you are thinking of _contributing_ to this repository (hooray! 🎉),
```

File: README.md [hunk 2]
```diff
@@ -1,5 +1,27 @@
...
+please make sure you
+[read the `Contributing` section](https://github.com/dwyl/quotes#contributing)
+of this repo.  
+
+There are many websites that already contain quotes and this repository is
+intended as a _sub-section_ of uniquely _inspiring_ quotes.
```
Comment (nelsonic) [inline README.md:11]:
```suggestion
intended as a _sub-set_ of uniquely _inspiring_ quotes.
```

File: README.md [hunk 3]
```diff
@@ -1,5 +1,27 @@
...
+See [`Avoid Quotes That Don't Inspire Action`](https://github.com/dwyl/quotes#avoid-quotes-that-dont-inspire-action).
+
+As such, ***we remind you that we will only be able to accept PRs that conform
+to the following key criteria***:
+1. An issue has been opened to _propose_ the quote, including why _you_ find it
+particularly inspiring :fire:
```
Comment (nelsonic) [inline README.md:17]:
```suggestion
particularly inspiring 💡
```

File: README.md [hunk 4]
```diff
@@ -1,5 +1,27 @@
...
+2. That issue has received _at least_ 2 'thumbs up' (:+1:) from people in the
+dwyl community.
+3. Your PR refers to the issue itself and has a clear title and description.
```
Comment (nelsonic) [inline README.md:20]:
```suggestion
3. Your PR refers to the issue itself and has a clear title and description. Not "Updates Quotes".
```

File: README.md [hunk 5]
```diff
@@ -1,5 +1,27 @@
...
+
+> Thank you again for your contributions to this and [other dwyl repos](https://github.com/dwyl/?q=learn_),
```
Comment (nelsonic) [inline README.md:22]:
```suggestion
> Thank you again for your contributions to this and [other dwyl repos](https://github.com/dwyl/?q=learn),
```

**Review by nelsonic — COMMENTED** [review]
@iteles looks good. 👍 
I've made a few suggestions for improvement. 
Please confirm/reject so we can merge.
Thanks.

File: README.md (hunk 1 above)
Comment (iteles) [reply README.md:5]: I didn't think that adding the markdown would make a difference here! Thanks for the update!

Comment (nelsonic) [reply README.md:5]: Some browsers / markdown parsers don't automatically display emoji, so I try to use the actual unicode character wherever possible. 👍

**Review by nelsonic — APPROVED** [review]
@iteles thanks. 🚀

**Comment by iteles** [comment]
These are all great suggestions, thanks @nelsonic!

Committed and can be merged once Travis passes 👍 😊

**Comment by nelsonic** [comment]
image

**Comment by nelsonic** [comment]
2urG

**Pull request merged**